app/commands/donations/stripe/sync_subscriptions.rb

```diff
@@ -0,0 +1,48 @@
+# Ensure that our local Stripe subscriptions match the actual Stripe subscriptions
+class Donations::Stripe::SyncSubscriptions
+  include Mandate
+
+  def call
+    create_subscriptions!
+    deactivate_subscriptions!
+  end
+
+  def create_subscriptions!
+    missing_active_subscription_ids = active_stripe_subscription_ids - local_subscription_ids
+    missing_active_subscription_ids.each do |subscription_id|
+      subscription = active_stripe_subscriptions[subscription_id]
+
+      user = User.find_by(stripe_customer_id: subscription.customer)
+      next unless user
+
+      Donations::Subscription::Create.(user, subscription)
+    end
+  end
```

Comment:
This will create a subscription for any active subscription in Stripe that is not yet in our database.
One caveat: this only works if the user already has a stripe customer ID.